ocs_ci/ocs/resources/pvc.py

```diff
@@ -130,3 +130,30 @@ def create_multiple_pvc(number_of_pvc=1, pvc_data=None):
         pvc_objs.append(pvc_obj)
         log.info(f'Created Persistent Volume Claim {pvc_name}')
     return pvc_objs
+
+
+def get_pv_for_pvc(pvc_obj):
+    """
+    Function to return a PV object for the given PVC
+
+    Args:
+        pvc_obj (OCS): OCS object representing PVC
+
+    Returns:
+        pv_obj (OCS): A OCS object for PV
+    """
+    pvc_obj.reload()
+    namespace = pvc_obj.namespace
+    pv_name = pvc_obj.data.get('spec').get('volumeName')
```

Comment:
As suggested in #368, PVC object can be used here and then pvc_obj.backed_pv to get the name of that PV